backend/oasst_backend/api/v1/utils.py

```diff
@@ -4,6 +4,9 @@
 from oasst_backend.models import Message
 from oasst_shared.schemas import protocol
 
+EMPTY_LIST = []
+EMPTY_DICT = {}
```

Comment:
dangerous because stateful. would rather just write `[]` and `{}` inline, even if we are "sure" we'll never mutate them

Reply:
OK, I can remove it .. otherwise hundreds of thousands of empty objects may be created per request (only to have fallback objects). But if you think that's no problem I can remove these.

Reply:
removed the globals